src/pages/billing/List.js

```diff
@@ -47,6 +47,8 @@ const styles = {
   }
 }
 
+const isCreatingStatus = status => _.includes(status, ['Creating', 'CreatingLandingZone'])
```

Comment:
In reality, we are only going to use `CreatingLandingZone` for the Azure case, but `Creating` is a status that exists so it seems reasonable to handle it if it gets used in the future.